types/webrtc/RTCPeerConnection.d.ts

```diff
@@ -141,7 +141,7 @@ interface RTCRtpCodecCapability {
 
 // https://www.w3.org/TR/webrtc/#idl-def-rtcrtpheaderextensioncapability
 interface RTCRtpHeaderExtensionCapability {
-    uri?: string | undefined;
```

Comment:
Do things fail if this is just changed to `uri: string`?

The spec linked above says that it can't be undefined, so it might be "correct" to just make it required.

Reply:
it fails in 5.0

Reply:
Dang, I was hoping it was just a new type so these wouldn't have to be duplicated. Oh well.